### DISCUSSION TITLE
Zugriff verboten - CSRF check failed

### DISCUSSION THREAD

**Discussion by GUNT0815** [nextcloud/all-in-one#3870] [Questions] [answered]
I am using the lastest nextcloud-aio image on an unraid server with caddyV2 as reverse proxy. All configured as recommended on github.

I receive "Zugriff verboten - CSRF check failed" messages repeatedly. This is impacting the usability a lot. I checked the internet already to find a solution, but was unsuccessful. 

Do I have to add something to the caddyfile in order to get read of the message ....?

------------Caddyfile--------------
https://mycloud.de:443 {
	reverse_proxy localhost:11000
}
------------------------------------

Addition:

I have seen the following error message referring to redis:

{"reqId":"OO7vJEdsRoho4O06m44P","level":4,"time":"2023-12-04T07:42:00+00:00","remoteAddr":"","user":"--","app":"no app in context","method":"","url":"--","message":"Could not boot workflowengine: php_network_getaddresses: getaddrinfo for nextcloud-aio-redis failed: Name does not resolve","userAgent":"--","version":"27.1.4.1","exception":{"Exception":"RedisException","Message":"php_network_getaddresses: getaddrinfo for nextcloud-aio-redis failed: Name does not resolve","Code":0,"Trace":[{"file":"/var/www/html/lib/private/RedisFactory.php","line":137,"function":"pconnect","class":"Redis","type":"->","args":["nextcloud-aio-redis",6379,0,null,0,0]},{"file":"/var/www/html/lib/private/RedisFactory.php","line":178,"function":"create","class":"OC\\RedisFactory","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/html/lib/private/Memcache/Redis.php","line":66,"function":"getInstance","class":"OC\\RedisFactory","type":"->","args":[]},{"file":"/var/www/html/lib/private/Memcache/Redis.php","line":72,"function":"getCache","class":"OC\\Memcache\\Redis","type":"->","args":[]},{"file":"/var/www/html/apps/workflowengine/lib/Manager.php","line":160,"function":"get","class":"OC\\Memcache\\Redis","type":"->","args":["events"]},{"file":"/var/www/html/apps/workflowengine/lib/AppInfo/Application.php","line":73,"function":"getAllConfiguredEvents","class":"OCA\\WorkflowEngine\\Manager","type":"->","args":[]},{"file":"/var/www/html/lib/private/AppFramework/Bootstrap/FunctionInjector.php","line":66,"function":"registerRuleListeners","class":"OCA\\WorkflowEngine\\AppInfo\\Application","type":"->","args":[["OC\\EventDispatcher\\EventDispatcher"],["OC\\AppFramework\\DependencyInjection\\DIContainer"],["OC\\AppFramework\\ScopedPsrLogger"]]},{"file":"/var/www/html/lib/private/AppFramework/Bootstrap/BootContext.php","line":50,"function":"injectFn","class":"OC\\AppFramework\\Bootstrap\\FunctionInjector","type":"->","args":[["Closure"]]},{"file":"/var/www/html/apps/workflowengine/lib/AppInfo/Application.php","line":65,"function":"injectFn","class":"OC\\AppFramework\\Bootstrap\\BootContext","type":"->","args":[["Closure"]]},{"file":"/var/www/html/lib/private/AppFramework/Bootstrap/Coordinator.php","line":200,"function":"boot","class":"OCA\\WorkflowEngine\\AppInfo\\Application","type":"->","args":[["OC\\AppFramework\\Bootstrap\\BootContext"]]},{"file":"/var/www/html/lib/private/App/AppManager.php","line":437,"function":"bootApp","class":"OC\\AppFramework\\Bootstrap\\Coordinator","type":"->","args":["workflowengine"]},{"file":"/var/www/html/lib/private/App/AppManager.php","line":216,"function":"loadApp","class":"OC\\App\\AppManager","type":"->","args":["workflowengine"]},{"file":"/var/www/html/lib/private/legacy/OC_App.php","line":126,"function":"loadApps","class":"OC\\App\\AppManager","type":"->","args":[[]]},{"file":"/var/www/html/lib/private/Console/Application.php","line":121,"function":"loadApps","class":"OC_App","type":"::","args":[]},{"file":"/var/www/html/console.php","line":99,"function":"loadCommands","class":"OC\\Console\\Application","type":"->","args":[["Symfony\\Component\\Console\\Input\\ArgvInput"],["Symfony\\Component\\Console\\Output\\ConsoleOutput"]]},{"file":"/var/www/html/occ","line":11,"args":["/var/www/html/console.php"],"function":"require_once"}],"File":"/var/www/html/lib/private/RedisFactory.php","Line":137,"message":"Could not boot workflowengine: php_network_getaddresses: getaddrinfo for nextcloud-aio-redis failed: Name does not resolve","exception":[],"CustomMessage":"Could not boot workflowengine: php_network_getaddresses: getaddrinfo for nextcloud-aio-redis failed: Name does not resolve"},"id":"656dcf2a0e796"}

**Comment by szaimen** [comment, marked as answer]
Hi, this is caused by https://github.com/nextcloud/all-in-one/discussions/2557
